internal/services/pubkey_service_test.go

```diff
@@ -9,29 +9,19 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	dao_stubs "github.com/RHEnVision/provisioning-backend/internal/dao/stubs"
+	"github.com/RHEnVision/provisioning-backend/internal/dao/stubs"
 	"github.com/RHEnVision/provisioning-backend/internal/models"
 	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
 	"github.com/stretchr/testify/assert"
 )
 
-func buildPkStore() []*models.Pubkey {
-	return []*models.Pubkey{&models.Pubkey{
-		ID:        1,
-		AccountID: 2,
-		Name:      "firstkey",
-		Body:      "sha-rsa body",
-	}, &models.Pubkey{
-		ID:        2,
-		AccountID: 4,
-		Name:      "secondkey",
-		Body:      "sha-rsa body",
-	}}
-}
-
 func TestListPubkeysHandler(t *testing.T) {
 	ctx := identity.WithIdentity(t, context.Background())
-	ctx = dao_stubs.WithPubkeyDao(ctx, buildPkStore())
+	ctx = stubs.WithPubkeyDao(ctx)
+	_, err := stubs.GeneratePubkey(ctx, nil)
+	assert.Nil(t, err, fmt.Sprintf("Error while generating a pubkey: %v", err))
+	_, err = stubs.GeneratePubkey(ctx, nil)
```

Comment:
Why have you chosen to send nil and create pubkey this way?

Reply:
On the other hand not allowing the shorthand would simplify the return to just error, which would be neat :thinking:

Reply:
I think it is cleaner to pass an "empty" pub key

Reply:
oki :+1: :)